Packs/MongoDB/Integrations/MongoDB/MongoDB.py

```diff
@@ -73,37 +78,70 @@ def normalize_id(entry: dict):
         Examples:
             >>> Client.normalize_id({'_id': ObjectId('5e4412f230c5b8f63a7356ba')})
             {'_id': '5e4412f230c5b8f63a7356ba'}
-        '''
+        """
         entry['_id'] = str(entry.pop('_id'))
         return entry
 
+    @classmethod
+    def datetime_to_str(cls, obj):
```

Comment:
add type hints

Reply:
It'll get any and will return any... but fine